README.md

```diff
@@ -73,20 +73,29 @@ Install [Scanitizer](https://github.com/apps/scanitizer) (currently in Beta) on
 ```yaml
 - uses: azure/container-scan@v0
   with:
-    image-name: my-image
+    image-name: my-image:${{ github.sha }}
 ```
 
 ### Container scan of an image available on a private registry
 
 ```yaml
 - uses: azure/container-scan@v0
   with:
-    image-name: loginServerUrl/my-image # loginServerlUrl would be empty if it's hosted on dockerhub
+    image-name: loginServerUrl/my-image:${{ github.sha }} # loginServerlUrl would be empty if it's hosted on dockerhub
     username: ${{ secrets.DOCKER_USERNAME }}
     password: ${{ secrets.DOCKER_PASSWORD }}
 ```
 
-## End to end workflow
+### Container scan of an image available locally, publically, or privately using workflow environment variables
+```yaml
+- uses: azure/container-scan@v0
+  with:
+    image-name: ${{ env.loginServerUrl }}/my-image:${{ github.sha }} # loginServerlUrl would be empty if it's hosted on dockerhub

```

Comment:
In case of dockerhub the `image-name` would be `/my-image` right? That's not a valid image name. 
You may want to edit your comment

Reply:
Yes, in the case of DockerHub, the `image-name` would be `my-image` - why is that not a valid image? If it's because of the `/`, I have updated the comments for all 3 examples. The main purpose with this 3rd example is to show how you could use login server as an environment variable (again, this requires double braces) with the task.